File: sdk/nodejs/examples/quickstart.ts
import { AgentKernel } from "agentkernel";

const client = new AgentKernel();

// Health check
console.log("Health:", await client.health());

// Run a command
const result = await client.run(["echo", "Hello from agentkernel!"]);
console.log("Output:", result.output);

// List sandboxes
const sandboxes = await client.listSandboxes();
console.log("Sandboxes:", sandboxes);

File: sdk/rust/examples/quickstart.rs
use agentkernel_sdk::AgentKernel;

#[tokio::main]
async fn main() -> agentkernel_sdk::Result<()> {
    let client = AgentKernel::builder().build()?;

    // Health check
    let status = client.health().await?;
    println!("Health: {status}");

    // Run a command
    let result = client
        .run(&["echo", "Hello from agentkernel!"], None)
        .await?;
    println!("Output: {}", result.output);

    // List sandboxes
    let sandboxes = client.list_sandboxes().await?;
    println!("Sandboxes: {sandboxes:?}");

    Ok(())
}
